app/forecast/forecast.html

```diff
@@ -3,12 +3,12 @@
     <a href="#/">Back to Search</a>
   </p>
 
-  Forecast for {{ vm.city }}
+  Forecast for {{ vm.cityService.city }}
   <hr />
   Days: 
-  <a href="#/forecast/2" ng-class="{ 'bg-primary': days == '2' }">2</a> |
-  <a href="#/forecast/5" ng-class="{ 'bg-primary': days == '5' }">5</a> |
-  <a href="#/forecast/7" ng-class="{ 'bg-primary': days == '7' }">7</a>
+  <a ui-sref="forecastDays({days: 2})" ng-class="{ 'bg-primary': days == '2' }">2</a> |
```

Comment:
In javascript, as a best practice, use `===` and `!==` instead of `==` and `!=` unless there's a specific use case, at which point, leave a comment, explaining why coercive equality check is needed.